packages/blockchain-wallet-v4-frontend/src/components/Flyout/TermsAndConditions/index.tsx

```diff
@@ -126,22 +117,13 @@ const TermsAndConditions: React.FC<InjectedFormProps<{}, Props> & Props> = (prop
             </HeaderTitle>
           </HeaderWrapper>
         </Header>
-        <Content mode='middle'>
+        <Content mode='top'>
           <PagePlaceholder>
-            <Markdown
-              options={{
-                overrides: {
-                  h1: {
-                    component: StyledHeader
-                  },
-                  p: {
-                    component: StyledParagraph
-                  }
-                }
-              }}
-            >
-              {props.termsAndConditions.termsAndConditions}
-            </Markdown>
+            <TextWrapper>
+              <TextBox
+                dangerouslySetInnerHTML={{ __html: props.termsAndConditions.termsAndConditions }}
```

Comment:
I know this data is coming from our own endpoint so there's minimal risk here but just to be safe think we should wrap this in DOM purify https://www.npmjs.com/package/dompurify